src/pybind/mgr/dashboard/rest_client.py

```diff
@@ -465,6 +476,12 @@ def do_request(self,
             logger.exception("%s REST API failed %s: %s", self.client_name,
                              method.upper(), str(ex))
             raise RequestException(str(ex))
+        except Timeout as ex:
+            msg = "{} REST API {} timed out after {} seconds (url={}).".format(
+                self.client_name, ex.request.method, Settings.REST_REQUESTS_TIMEOUT,
+                ex.request.url)
```

Comment:
Can we make use of `str(ex)` here?

Reply:
The default error message does not provide all that information: 
``HTTPConnectionPool(host='dev-vtheile-ceph-1.oa.suse.de', port=8000): Read timed out. (read timeout=3)``